server/src/database/migrations/20201013161131_users_visits.js

```diff
@@ -2,18 +2,13 @@ exports.up = function (knex) {
   return knex.schema.createTable('users_visits', (table) => {
     table.increments();
     table.timestamp('created_at').defaultTo(knex.fn.now());
-    table.string('username').notNullable();
-    table.integer('visit_id').unsigned().notNullable();
-
-    table.unique([ 'username', 'visit_id' ]);
+    table.timestamp('visited_at').defaultTo(knex.fn.now());
     table
-      .foreign('username')
+      .string('username').notNullable()
       .references('username')
-      .inTable('users');
     table
-      .foreign('visit_id')
-      .references('id')
-      .inTable('visits');
+      .string('visited_username').notNullable()
+      .references('username')
```

Comment:
Ah ok je vois ce que tu as voulu faire mais c'est pas bon.
Le ".references" il faut le remettre comme avant mais à part de la création de colonne.

```suggestion
      table.foreign('visited_username').references('username').inTable('users')
```